Week01/Problem02/asedler_02.py

```diff
@@ -0,0 +1,16 @@
+"""
+This script implements a solution to Project Euler Problem 02
+by summing the even-valued terms of the Fibbonacci sequence
+that are less than 4 million.
+
+@author: Andrew Sedler
+"""
+
+fib = [1,2]
+
+while fib[-1] < 4000000:
+    fib.append(fib[-2] + fib[-1])
```

Comment:
Your solution here works, but it can be dangerous to create very long lists (imagine if we were checking Fibbonaci terms below 1e50!) especially from within Python, which sorta plays fast and loose with system memory.  Instead of storing the Fibbonaci numbers, you could store the previous 2 terms, create the next term, check if it's even, and add it to an accumulation variable:
```
even_total = 0
while current < 4000000:
    next = current + previous
    previous = current
    current = next
    if current % 2: 
        even_total += current
```